skimage/transform/_geometric.py

```diff
@@ -612,6 +612,20 @@ def __call__(self, coords):
         coords_out : (N, D) array
             Destination coordinates.
 
+        Examples
+        --------
+        The call method enables the class instance
+        to behave like functions. The user can call
+        these instances like functions.
+
+        matrix=[[1 2 3]
+                [4 5 6]
+                [7 8 9]].
+        
+        A=ProjectiveTransform(matrix)  #instance created.
+        B=A(coords)    #calling instance A as a function
```

Comment:
```suggestion
        B = A(coords) # calling instance A as a function
```

Reply:
Unresolving conversation because I don't see the output of B anywhere...